entrypoint.sh

```diff
@@ -18,15 +18,9 @@ case "$INPUT_SCRIPT" in
     break
     ;;
   *)
-    SCRIPT_PATH="$GITHUB_WORKSPACE/$INPUT_SCRIPT"
-    SANS_ARGS=$(echo "$SCRIPT_PATH" | cut -d' ' -f1)
-    if test -z "$GITHUB_WORKSPACE" || test ! -f "$SANS_ARGS"; then # Ensure script exists
```

Comment:
I still like these checks, can we keep them?

Reply:
Although I now remember it errored out in a wrong way before. Let's remove it sorry!

Reply:
@Kyubinhan These checks are at odds with setting env vars before running the script. We can choose between arbitrary *sh commands or a script file. 

I'm not sure these checks are super helpful. Without the checks and if pointing to a script that doesn't exist, *sh should return `command not found: [YOUR SCRIPT]` which is equally helpful. Since `set +x` is set, it will halt there, returning the error code.

If you can think of a way to support arbitrary commands as well as checking for the file, feel free to push!

Reply:
For example `myscript.sh myarg` sets `SANS_ARGS` to `myscript.sh`. Checking for that file succeeds. However, `VAR="hello" && myscript.sh myarg` sets `SANS_ARGS` to `VAR="hello"` which does not pass the file check.

Reply:
👍